### ISSUE TITLE
Compendium & Inventory Rolled Back to 1-2 Days Prior (US-WEST PVE)

### ISSUE THREAD

**Issue by Kaelah** [Uuvana-Studios/longvinter-windows-client#1198]
I had completed the Fish and Feather sections of the compendium so all vendors were unlocked... When I signed in today, both had been reset to my progress from a couple days prior so now I only have the plant section unlocked. My character inventory has also reset to a previous version where I had a vending machine in it, even though it had been placed and the placed one is still there - so now I have an extra one.

All items from my house that were added are still there (including completed house upgrades, placed storage boxes, etc.), but my compendium & inventory rolled back.

**To Reproduce**
1. Complete compendium.
2. Sign out and go to sleep.
3. Next day the progress was reset or rolled back & inventory is changed to an older set of items.

**Expected behavior**
Compendium progress & inventory should save and be there the next day.

**Desktop (please complete the following information):**
 - OS: Win10 Pro
 - Game Version 1.0.8b
 - Server: US-WEST PVE

**Comment by GemsLikeTaric** [comment]
I'm having this issue today. My house upgrades are gone and half of my property is missing. Also, none of the changes that I made to my property 3 days ago are there. (e.g. new fence type installed, has been rolled back to previous fence)

**Comment by Kaelah** [comment]
> I'm having this issue today. My house upgrades are gone and half of my property is missing. Also, none of the changes that I made to my property 3 days ago are there. (e.g. new fence type installed, has been rolled back to previous fence)

Today my house rolled back, too. My friend on the same server put some wood in my house, I signed in about ~30 mins after they got online and when I spawned in my lot rolled back so now I have 0 upgrades (had the electricity transmitter + wind turbine) and the second vending machine I had placed isn't there anymore.... lost the items I put into it. Prior to me signing in, my friend saw the second vending machine still there.

I guess I'll just play Tiny Tina's Wonderlands + Alice: Madness Returns instead lol.
